src/common/utils/oidc/secret.go

```diff
@@ -98,7 +80,18 @@ func (dm *defaultManager) VerifySecret(ctx context.Context, userID int, secret s
 	if secret != plainSecret {
 		return verifyError(errors.New("secret mismatch"))
 	}
-	tokenStr, err := utils.ReversibleDecrypt(oidcUser.Token, key)
+	return dm.VerifyToken(ctx, oidcUser)
+}
+
+func (dm *defaultManager) VerifyToken(ctx context.Context, user *models.OIDCUser) error {
```

Comment:
please add some comments for VerifyToken, and its name is same as helper.VerfiyToken.

It confuses code reader.

Reply:
I'll add some comment.
But actually there's no need to add comment because it's a func of a struct that is not exposed.